packages/rocketchat-ui/client/views/app/directory.html

```diff
@@ -1,87 +1,92 @@
 <template name="directory">
 	<section class="rc-directory">
 		{{# header sectionName="Directory" hideHelp=true}}
-		<div class="rc-directory-header">
-		<select class="rc-input__element rc-directory-selector js-typeSelector" name="type">
-				<option value="channels">{{_ 'Channels'}}</option>
-				<option value="users">{{_ 'Users' }}</option>
-			</select>
-			<div class="rc-input rc-input--small rc-directory-search">
-				<label class="rc-input__label">
-					<div class="rc-input__wrapper">
-						{{> icon icon="magnifier" block="rc-input__icon" }}
-						<input type="text" class="rc-input__element rc-input__element--small js-search" name="message-search" id="message-search" placeholder="{{_ 'Directory_Search_Placeholder'}}" autocomplete="off">
-					</div>
-				</label>
+			<div class="rc-directory-header">
+				<select class="rc-input__element rc-directory-selector js-typeSelector" name="type">
+					<option value="Channels">{{_ 'Channels'}}</option>
+					<option value="users">{{_ 'Users' }}</option>
+					<option value="Private_Groups">{{_ 'Private' }}</option>
+					{{#each roomType in customRoomTypes}}
+						<option value={{roomType}}> {{roomType}}</option>
```

Comment:
`{{roomType}}` should be an i18n-supported text. I didn't see this in the helper. At my client, the dropdown value was not translated

Reply:
hanlded.